Android/app/src/main/java/io/github/project_travel_mate/utilities/DailyQuotesFragment.java

```diff
@@ -171,10 +171,13 @@ private static Bitmap getScreenShot(View view) {
     }
 
     void doNotShowClicked() {
+        /*
```

Comment:
Instead of commenting this code, just remove it.

Reply:
Well, did you check why the code was not going inside this if?
which condition was not getting satisfied?